config/locales/en.yml

```diff
@@ -20,4 +20,117 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
```

Comment:
@miurahr Ping on this if you have any ideas. I've seen that previously in other frameworks.